### PR TITLE
added user logging roles

### PR DESCRIPTION
**Pull request by paynejacob**
Added cluster roles for users:

logging:admin
Flow: CRUD+L+DL+W
Output: CRUD+L+DL+W

logging:edit
Flow: CRUD+L+DL+W
Output: CRUD+L+DL+W

logging:view
Flow: RL+W
Output RL+W

### CODE REVIEW

File: packages/rancher-logging/overlay/templates/userroles.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: "logging:admin"
```
Comment (cbron) [inline packages/rancher-logging/overlay/templates/userroles.yaml]:
```suggestion
  name: "logging-admin"
```
Same for below

File: packages/rancher-logging/overlay/templates/userroles.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups:
+      - "flows.logging.banzaicloud.io"
+      - "outputs.logging.banzaicloud.io"
+    resources:
+      - Flow
```
Comment (cbron) [inline packages/rancher-logging/overlay/templates/userroles.yaml]: Have resources be lower cased and plural. [Examples](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-examples)

File: packages/rancher-logging/overlay/templates/userroles.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      - Output
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+      - deletecollection
```
Comment (cbron) [inline packages/rancher-logging/overlay/templates/userroles.yaml]: This works, but I wonder if we should just give them `*` ?

**Review by brendarearden — APPROVED** [review]
L🔥 TM

**Review by cbron — APPROVED** [review]

**Pull request merged**